source/blog/2018-07-13-the-ember-times-issue-55.md

```diff
@@ -36,9 +36,22 @@ Check out what's been going in Emberland 🐹 these days:
 
 ---
 
-## [SECTION TITLE](#section-url)
-
-
+## [Find Ember on the Interwebs with the Power of SEO 🔎](#section-url)
+
+These days, weeks and months lots of efforts have been focussing on improving the
```

Comment:
Would rewrite this as:
These `past` days, weeks and months ... been `focused` on...